test/run-pass/language-demonstration/spoj-programs/check-pesel.ieo

```diff
@@ -0,0 +1,38 @@
+# - run:
+#     stdin:
+#       - "2"
+#       - "44051401458"
+#       - "12345678901"
+#     stdout: 
+#       - "D"
+#       - "N"
+
+# https://pl.spoj.com/problems/JPESEL/
```

Comment:
nitpicking: I would prefer to refer to English SPOJ instead of Polish one because we are targeting an international audience